ansys/mapdl/core/launcher.py

```diff
@@ -146,8 +146,26 @@ def port_in_use(port, host=LOCALHOST):
             return True
 
 
+def delete_ip_file(path):
+    """Delete the created 'mylocal.ip'."""
+    # path=''
+    file_name = os.path.join(path, 'mylocal.ip') 
+    if os.path.exists(file_name):
+        # os.remove(file_name)
+        pass 
```

Comment:
```suggestion
    file_name = os.path.join(path, 'mylocal.ip') 
    if os.path.exists(file_name):
        os.remove(file_name)
```

Reply:
I wonder if it is a good practice to leave the file around. I understand we cannot do much at the time because of the delay in ansys to read that `mylocal.ip` file. 

If you want I can add a small function inside `mapdl.exit()` which checks if that file exist and delete it.  You know what? I'm going to do it. Just let me know if you prefer not to.

Reply:
Done. See commit #3a6b312a221aa582b33da90a7d038b7d611370b5